src/components/change-requests/review-change-request/review-change-request/review-change-request.test.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { exampleStandardChangeRequest } from "../../../../test-support/test-data/change-requests.stub";
+import { render, screen } from "@testing-library/react";
```

Comment:
you'll actually need to import from one of our utils files because we did some configuring to modify the render function

https://github.com/Northeastern-Electric-Racing/PM-Dashboard-v2/blob/1121297ac2263c7c55085c8faef3fd2a759f0ed7/src/components/change-requests/change-request-details/change-request-details.test.tsx#L9

Reply:
Where would I use `routerWrapperBuilder` in my tests? Most of my tests in this file are for making sure the component renders correctly so I don't really know where I would use this so that I don't get the "declared but never used" error from the linting tools. Is this something I would wrap around my component in the `renderComponent` function?

Reply:
You want to [use it like this](https://github.com/Northeastern-Electric-Racing/PM-Dashboard-v2/blob/1121297ac2263c7c55085c8faef3fd2a759f0ed7/src/components/change-requests/change-request-details/change-request-details.test.tsx#L30) when you don't need to control the navigation URL. You want to [use it like this](https://github.com/Northeastern-Electric-Racing/PM-Dashboard-v2/blob/1121297ac2263c7c55085c8faef3fd2a759f0ed7/src/components/change-requests/change-requests.test.tsx#L41) when you do need to control the navigation URL. In either case, it gets used inside of the test file's `renderComponent` helper function.

Can you share a screenshot of the error with the ternary expression? I'm not sure what you mean. We ideally want to stick with importing from `test-utils` and not directly from the package.

Reply:
Here's the screenshot of the error:
![failed-test-cr-review](https://user-images.githubusercontent.com/48561685/148670833-f45ab83a-af53-49c2-96ee-59ea26389687.png)

Here's a screenshot of the line of code that I think is causing the issue:
![possible-test-failure-src](https://user-images.githubusercontent.com/48561685/148670851-461a9505-0e5e-487f-ba3d-6c59b4c804d9.png)